src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__length--28 calendar__day--fri">
```

Comment:
You should set padding acording to this point 
`Use flex with 1px gap and limit its width to exactly 7 columns + 10px paddings` 
of [checklist](https://github.com/mate-academy/layout_calendar)

Think in what block you could add it.